documentation/bolt_running_tasks.md

```diff
@@ -3,38 +3,53 @@
 Bolt can run Puppet tasks on remote targets without requiring any Puppet
 infrastructure. 
 
-To execute a task, run `bolt task run`, specifying:
+To execute a task, specify the following:
 
 -   The full name of the task, formatted as `<MODULE::TASK>`, or as `<MODULE>`
     for a module's main task (the `init` task).
 -   Any task parameters, as `parameter=value`.
 -   The targets on which to run the task and the connection protocol, with the
-    `--targets` flag.
+    `targets` command-line option.
 -   If credentials are required to connect to the target, the username and
-    password, with the `--user` and `--password` flags.
+    password, with the `user` and `password` command-line options.
 
 For example, to run the `sql` task from the `mysql` module on a target named
 neptune:
 
-```
-bolt task run mysql::sql database=mydatabase sql="SHOW TABLES" --targets neptune --modulepath ~/modules
-```
+- _\*nix shell command_
+
+  ```shell
+  bolt task run mysql::sql database=mydatabase sql="SHOW TABLES" --targets neptune --modulepath ~/modules
```

Comment:
Is the inclusion of `modulepath` here relevant to the example or is that a legacy from all the way back before we had a config file?

Reply:
Definitely legacy.